src/content/docs/merchants/get-started/multistore.mdx

```diff
@@ -125,8 +155,8 @@ These steps show you how to add a new English Canadian store view (`en_ca`) simi
 Create an `en_ca` store view in the Commerce Admin and perform the following steps in your project.
 
 <Steps>
-1. Create the `en_ca` folder under the project root in Sharepoint or Google Docs.
-1. Copy the configuration files, placeholders file, analytics file, and document pages from the `en` directory to the `en_ca` directory.
+1. Create the `/en_ca/` folder under the project root in SharePoint or Google Drive. You can name this folder anything you want, as long as it's unique and can be used in a web address.
```

Comment:
Add an example of a name that can't be used in a web address and show a corrected name that CAN be used.

Reply:
Done